src/scanoss/api/scanning/v2/scanoss_scanning_pb2_grpc.py

```
@@ -1,10 +1,30 @@
+_version_not_supported = False
+
+try:
+    from grpc._utilities import first_version_is_lower
+    _version_not_supported = first_version_is_lower(GRPC_VERSION, GRPC_GENERATED_VERSION)
+except ImportError:
+    _version_not_supported = True
+
+if _version_not_supported:
+    raise RuntimeError(
+        f'The grpc package installed is at version {GRPC_VERSION},'
+        + f' but the generated code in scanoss/api/scanning/v2/scanoss_scanning_pb2_grpc.py depends on'
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)